packages/in_app_purchase/in_app_purchase_storekit/example/lib/main.dart

```diff
@@ -22,7 +22,9 @@ void main() {
   runApp(_MyApp());
 }
 
-const bool _kAutoConsume = true;
+// Auto-consume must be true on iOS.
```

Comment:
Similarly, this whole variable can be removed here, and `autoConsume` hard-coded to true at the call site. This is just cruft from duplicating the example during federation.